internal/db/db.go

```
@@ -1,24 +1,27 @@
 package db
 
 import (
-	"github.com/Lerner17/shortener/internal/config"
-	filestorage "github.com/Lerner17/shortener/internal/db/file_storage"
-	"github.com/Lerner17/shortener/internal/db/memdb"
+	"github.com/Lerner17/shortener/internal/db/psql"
 	"github.com/Lerner17/shortener/internal/models"
 )
 
 type URLStorage interface {
 	GetURL(string, string) (string, bool)
 	CreateURL(string, string) (string, string)
 	GetUserURLs(string) models.URLs
+	CreateBatchURL(string, models.BatchURLs) (models.BatchShortURLs, error)
 }
 
 func GetDB() URLStorage {
-	cfg := config.GetConfig()
+	// cfg := config.GetConfig()
 
-	if cfg.FileStoragePath == "" {
-		return memdb.NewMemDB()
-	} else {
-		return filestorage.NewFileStorage(cfg.FileStoragePath)
-	}
+	// if cfg.DatabaseDsn != "" {
+	psql := psql.NewPostgres()
+	psql.Migrate()
```

Basically if migration is failed we should fail fast in order to avoid destruct of information